ubuntudesign/vanilla-framework
# Equal height audit

## Done

 - Prefixed the equal-height classes with "u-"
 - Removed a pixel measurement that should have been rem
 - Updated the equal-height documentation to use/reflect this

## QA

Run the vanillaframework.io project and check equal-heights in various viewports.

## Details

Fixes #407
